eduNEXT/tutor-contrib-edunext-distro
# test: fix tests and quality

This PR is to fix tests and quality checks for distro in favor of create CI worrkflows.

## Testing

You can create a new virtualenv, then:

1. run ``make python-test-requirements``
2. run ``make run-tests``

All tests should be passed properly.
